app/bingey-api/models/watchlist.js

```diff
@@ -0,0 +1,12 @@
+const { Schema, model } = require('mongoose');
+
+const watchlistSchema = new Schema(
+  {
+    titles: [{ titleId: { type: Schema.Types.ObjectId, ref: 'Titles' } }],
```

Comment:
I think reference should be for _Title_, the pluralization thingy again
```suggestion
    titles: [{ titleId: { type: Schema.Types.ObjectId, ref: 'Title' } }],
```